peertalk.xcodeproj/project.pbxproj

```diff
@@ -493,7 +493,7 @@
 			};
 			buildConfigurationList = 6A88FA2A150D613800FC3647 /* Build configuration list for PBXProject "peertalk" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
+			developmentRegion = en;
```

Comment:
Why no `Base` here?

Reply:
Because with a 3rd party library we don't want to be too far off from the original fork - base localization changes the way localization for xibs and storyboards is handled. If there are none of those, it's safe _for now_ but we can get problems in the future. If it's our own repo I rather use base localization as much as possible, even if it means doing a little conversion on the xibs/storyboards.

Reply:
Ok, I'll take that back as Xcode keeps showing a warning that it wants to switch on base localization, I'll switch it on and make a new PR - sigh 😞

Reply:
oh, and development region should always be en, not base - base should only be added to the knownRegions if we need it